.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I don't think we want to impose `.vscode` settings on the user, I'd prefer to remove the whole folder and add it to `.gitignore`.